addons/mrp/models/product.py

```diff
@@ -4,7 +4,16 @@
 from datetime import timedelta
 from odoo import api, fields, models
 from odoo.tools.float_utils import float_round, float_is_zero
+import operator as py_operator
```

Comment:
import operator should be below import datetime ([guidelines.html#imports](https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#imports))